components/Modal.jsx

```diff
@@ -0,0 +1,34 @@
+import { useRef, useEffect } from "react";
+import styles from "@/styles/components/modal.module.scss";
+
+export default function Modal({ content, isModalOpen, setIsModalOpen }) {
+	const modalRef = useRef(null);
+
+	useEffect(() => {
+		if (isModalOpen) {
+			modalRef.current.showModal();
+		}
+	}, [isModalOpen]);
+
+	const backgroundClicked = (event) => {
```

Comment:
think it would be better to call it `handleBackgroundClick` as it would be more consistent with other events you called before.

Reply:
noted